docs/vscode.md

```diff
@@ -0,0 +1,104 @@
+# Visual Studio Code Installation
+Install nodeJs, yarn and java according to your system:
+- Windows: [./windows-setup.md](./windows-setup.md)
+- Linux / Windows Subsystem for Linux: [./wsl-setup.md](./wsl-setup.md)
+
+Download and Install Visual Studio Code or use chocolatey:
+
+https://code.visualstudio.com/Download  
+https://code.visualstudio.com/docs/setup/linux  
```

Comment:
The documentation about Visual Studio code is independent from the operating system. 
As I prefer using apt-get instead of downloading the pure packages I thougt I add also a link to the linux install guide. 
But I'm not a linux guy, so perhaps the link to the main page is sufficient?